packages/php-wasm/web/src/lib/web-php.ts

```diff
@@ -44,6 +48,560 @@ const fakeWebsocket = () => {
 	};
 };
 
+async function generateKeys() {
```

Comment:
this and `createCertificate()` would both be very good functions to document, specifically why they exist, what kind of keys and certificates they generate, and why